test/system/src/test/resources/pravega.properties

```diff
@@ -14,4 +14,8 @@ bookkeeper.bkAckQuorumSize=3
 controller.transaction.maxLeaseValue=120000
 controller.retention.frequencyMinutes=2
 log.level=DEBUG
-hdfs.replaceDataNodesOnFailure=false
\ No newline at end of file
+hdfs.replaceDataNodesOnFailure=false
+#4GB of cache.
+pravegaservice.cacheMaxSize=4294967296
+#4GB of cache + a buffer for whatever else Netty might need it for.
+io.netty.maxDirectMemory=5368709120
```

Comment:
How do we estimate how much direct memory might need? How did you do it here?

Reply:
Cache Size + 1GB. 

Whatever is in here doesn't matter. All of these are overridable via the manifests we deploy with, which have much more tailored values for the HW config we're deploying to.